packages/tracing/src/browser/web-vitals/getUpdatedCLS.ts

```diff
@@ -43,7 +43,8 @@ export const getUpdatedCLS = (onReport: ReportHandler, reportAllChanges?: boolea
 
   const entryHandler = (entry: LayoutShift): void => {
     // Only count layout shifts without recent user input.
-    if (!entry.hadRecentInput) {
+    // TODO: Figure out why entry can be undefined
+    if (entry && !entry.hadRecentInput) {
```

Comment:
@AbhiPrasad you'll also want to check if `sessionEntries` is a non-empty array on line 54. `lastSessionEntry` could be undefined as well